src/scalarstats.jl

```diff
@@ -240,6 +312,15 @@ function _zscore_chksize(X::AbstractArray, μ::AbstractArray, σ::AbstractArray)
     end
 end
 
+
+"""
+    zscore!([Z,] X, μ, σ)
+
+Compute the z-scores of an array `X` with mean `μ` and standard deviation `σ`.
+z-scores are the signed number of standard deviations above the mean that an
```

Comment:
"above the mean" sounds a bit confusing here: why not "from the mean"? Could also say "z-scores are centered-reduced scores giving/equal to..." or something like that.